test/utils/nextpage/employer/cis/uk/subcontractor/WhatTypeOfSubcontractorNextPageSpec.scala

```diff
@@ -0,0 +1,142 @@
+      "/business-account/add-tax/employer/cis/uk/subcontractor/sole-trader",
+      "no Enrolment"
+    )
+
+    val nextPageSoleTraderTwoEnrolmentValue =
+      (WhatTypeOfSubcontractor.SoleTrader, Enrolments(Set(saEnrolment, ctEnrolment)))
+
+    behave like nextPageWithEnrolments(
+      NextPage.whatTypeOfSubcontractor,
+      nextPageSoleTraderTwoEnrolmentValue,
+      WhatTypeOfSubcontractor.SoleTrader.toString,
+      "/forms/form/CIS301302/start#1",
+      "two Enrolments"
+    )
+
+    val nextPagePartnershipSAValue = (WhatTypeOfSubcontractor.Partnership, Enrolments(Set(saEnrolment)))
+
+    behave like nextPageWithEnrolments(
+      NextPage.whatTypeOfSubcontractor,
+      nextPagePartnershipSAValue,
+      WhatTypeOfSubcontractor.Partnership.toString,
+      "/forms/form/Construction-Industry-Scheme-register-your-partnership/new",
+      "SA Enrolment"
+    )
+
+    val nextPagePartnershipCTValue = (WhatTypeOfSubcontractor.Partnership, Enrolments(Set(ctEnrolment)))
+
+    behave like nextPageWithEnrolments(
+      NextPage.whatTypeOfSubcontractor,
+      nextPagePartnershipCTValue,
+      WhatTypeOfSubcontractor.Partnership.toString,
+      "/forms/form/Construction-Industry-Scheme-register-your-partnership/new",
+      "CT Enrolment"
+    )
+
+    val nextPagePartnershipNoValue = (WhatTypeOfSubcontractor.Partnership, Enrolments(Set()))
+
+    behave like nextPageWithEnrolments(
+      NextPage.whatTypeOfSubcontractor,
+      nextPagePartnershipNoValue,
+      WhatTypeOfSubcontractor.Partnership.toString,
+      "https://www.gov.uk/government/publications/construction-industry-scheme-partnership-registration-cis304",
+      "no Enrolment"
+    )
+
+    val nextPageLimitedCompanySAValue = (WhatTypeOfSubcontractor.LimitedCompany, Enrolments(Set(saEnrolment)))
+
+    behave like nextPageWithEnrolments(
+      NextPage.whatTypeOfSubcontractor,
+      nextPageLimitedCompanySAValue,
+      WhatTypeOfSubcontractor.LimitedCompany.toString,
+      "https://www.gov.uk/government/publications/construction-industry-scheme-company-registration-cis305",
+      "SA Enrolment"
+    )
+
+    val nextPageLimitedCompanyCTValue = (WhatTypeOfSubcontractor.LimitedCompany, Enrolments(Set(ctEnrolment)))
+
+    behave like nextPageWithEnrolments(
+      NextPage.whatTypeOfSubcontractor,
+      nextPageLimitedCompanyCTValue,
+      WhatTypeOfSubcontractor.LimitedCompany.toString,
+      "https://www.gov.uk/government/publications/construction-industry-scheme-company-registration-cis305",
+      "CT Enrolment"
+    )
+
+    val nextPageLimitedCompanyNoValue = (WhatTypeOfSubcontractor.LimitedCompany, Enrolments(Set()))
+
+    behave like nextPageWithEnrolments(
+      NextPage.whatTypeOfSubcontractor,
+      nextPageLimitedCompanyNoValue,
+      WhatTypeOfSubcontractor.LimitedCompany.toString,
+      "https://www.gov.uk/government/publications/construction-industry-scheme-company-registration-cis305",
+      "no Enrolment"
+    )
+
+    /*
```

Comment:
Why has this been commented out. If it is not needed then remove it. If it is needed than uncomment it.

Reply:
👍